src/deephaven/plugin/matplotlib/__init__.py

```diff
@@ -156,6 +162,8 @@ def _step(self, update, *args):
 
     def _draw_frame(self, framedata):
         data = {}
-        for column in self._columns:
-            data[column] = dhnp.to_numpy(self._table, [column])
+        for i, column in enumerate(self._columns):
+            data_col = _JDataAccessHelpers.getColumn(self._table.j_table, column)
+            data[column] = dhnp.column_to_numpy_array(self._column_defs[i], data_col.getDirect())
```

Comment:
@jmao-denver this feels kind of dirty to me - can we add a wrapper function in `numpy.py` to get a column of data directly?
Using `dhnp.to_numpy` returns a table of data grouped by row, which isn't what we want in this case.

Reply:
We discussed on a huddle, and you let me know about just doing `[:, 0]` for accessing the first column which is exactly what we want.